validator/validators/stress/stress_validator.go

```diff
@@ -28,38 +28,38 @@ var (
 	metricPluginBoundValue = MetricPluginBoundValue{
 		"1000": {
 			"statsd": {
-				"procstat_cpu_usage":   float64(4),
+				"procstat_cpu_usage":   float64(19),
 				"procstat_memory_rss":  float64(66500000),
 				"procstat_memory_swap": float64(0),
 				"procstat_memory_vms":  float64(818000000),
 				"procstat_memory_data": float64(95000000),
 				"procstat_num_fds":     float64(9),
-				"net_bytes_sent":       float64(7500),
-				"net_packets_sent":     float64(21),
+				"net_bytes_sent":       float64(100000),
+				"net_packets_sent":     float64(100),
 			},
 		},
 		"5000": {
 			"statsd": {
-				"procstat_cpu_usage":   float64(7),
-				"procstat_memory_rss":  float64(66500000),
+				"procstat_cpu_usage":   float64(120),
+				"procstat_memory_rss":  float64(130000000),
 				"procstat_memory_swap": float64(0),
 				"procstat_memory_vms":  float64(818000000),
-				"procstat_memory_data": float64(95000000),
-				"procstat_num_fds":     float64(9),
-				"net_bytes_sent":       float64(7500),
-				"net_packets_sent":     float64(21),
+				"procstat_memory_data": float64(130000000),
+				"procstat_num_fds":     float64(19),
+				"net_bytes_sent":       float64(524000),
+				"net_packets_sent":     float64(520),
 			},
 		},
 		"10000": {
 			"statsd": {
-				"procstat_cpu_usage":   float64(14),
-				"procstat_memory_rss":  float64(95000000),
+				"procstat_cpu_usage":   float64(200),
```

Comment:
Quite the increase from 14% to 200%.
Why?

Reply:
I increased from 1 minutes to 10 minutes. Therefore, every minutes it holds 10000 metrics and process it with the force flush interval of 60 before able to deliver it. Therefore, increasing the CPU Usage when aggregating with SEH, and holding more metrics in batch which increase value in Memory metrics